apps/accounts/signal_handlers.py

```diff
@@ -23,6 +23,9 @@
 
 @receiver(pre_save, sender=Account)
 def on_account_pre_save(sender, instance, **kwargs):
+    if getattr(instance, 'skip_history_when_saving', False):
+        return
+
     if instance.version == 0:
         instance.version = 1
     else:
```

Comment:
Please format your question properly:

@receiver(pre_save, sender=Account)

def on_account_pre_save(sender, instance, using, **kwargs):

If you need further assistance with this code, please ask a specific question related to it.